src/pspm_sf_dcm.m

```diff
@@ -29,6 +29,8 @@
 %    .dispwin:  display progress window (default 1)
 %  .dispsmallwin:
 %               display intermediate windows (default 0);
+%    .missing:  index of missing values to ignore
```

Comment:
For `pspm_dcm`, `missing` is a field of `model` correct? Should we do the same here?

Reply:
Indeed, `missing` is a field of `model` in `dcm`. The question I had was `sf_dcm` does not have a struct `model`. May I know if I should manually add a struct `model` in `sf_dcm`?

Reply:
> OK, this makes sense - no need to create a "model" variable here. Just wondering whether we should lave such data-fields out of "options". I have no strong opinion, it could be a field in options or an additional input argument. It would also have to be forwarded from pspm_sf, which is something to think about.

Thank you @dominikbach . Currently `.missing` as a field in `options` will be relatively more compatible since we need to adjust the formula if we would like to make `.missing` as an independent parameter. Moreover, `.missing` is not a mandatory field here thus it may be a little difficult to put `.missing` as an independent parameter. Since `options` will be transferred when `sf_dcm` is called, `.missing` will be therefore transferred at the same time.